src/analysis/modals/ComputeModal/AutopauseConfiguration.ts

```diff
@@ -22,7 +22,7 @@ export const AutopauseConfiguration = (props: AutopauseConfigurationProps): Reac
     autopauseRequired = false,
     autopauseThreshold,
     disabled = false,
-    maxThreshold = 999,
+    maxThreshold = undefined,
```

Comment:
Does this mean that in theory non Lite workspaces will be able to set a threshold that is higher than 999 minutes? I think that is fine, I just want to make sure I understand

Reply:
This was a workaround to avoid showing the default 999 maximum in the "Choose a duration ..." message.